stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux

```diff
@@ -0,0 +1,105 @@
+//Final working code as of August 11, 2020
+//Currently supports single field classification and binary data sets 
+
+import "system"
```

Comment:
I think you need to run `make generate` so that this code gets converted into some Go code to correctly include it into the Flux build. See the Flux README for instructions on how to do that.

Reply:
Based on the guidelines in CONTRIBUTING.md, I ran `go generate ./stdlib` since I'm only contributing .flux files. Is this along the lines of what you're thinking of?

Reply:
Yes, running the generate command will create some Go files that are effectively the contents of the contributed .flux files. Specifically the Go files contain the parsed AST of the flux files.

Reply:
Yeah just noticed that. Currently working on writing some tests!

Reply:
Quick q: So for the tests, I'm trying to use csv.from() and point to the path of my csv file, but my naiveBayes flux code is set up to run a from(bucket: myBucket) to get csv data. Is there a way to write to a bucket in flux and then be able to pass that bucket name as an argument to naiveBayes(...).